features/src/main/scala/com/salesforce/op/stages/SparkStageParam.scala

```diff
@@ -55,9 +66,11 @@ class SparkStageParam[S <: PipelineStage with Params]
    * Spark stage saving path
    */
   var savePath: Option[String] = None
+  @transient var sbc: Option[SparkBundleContext] = None
```

Comment:
should `sbc` and `localTransformer` be private and instead exposed over a getter?

Reply:
I am following the convention of savePath. This is the parameter itself - there are getters and setters for the path and transformers on the spark wrapped stage itself but not on the parameter...